dnscrypt-proxy/config.go

```diff
@@ -102,6 +104,7 @@ func newConfig() Config {
 		CacheNegMaxTTL:           600,
 		CacheMinTTL:              60,
 		CacheMaxTTL:              86400,
+		ConfigTTL:                600,
```

Comment:
Having this knob is great!

But `configTTL` is not a very self-explanatory name. It doesn't expire a configuration :)
Can you think of a better name for it?

Reply:
I checked some others,

unbound has infra-host-ttl
dnsmasq has local-ttl, auth-ttl, neg-ttl

Maybe pluginsState.action == PluginsActionReject case:
refused_ttl / reject_ttl / blocked_ttl / neg_ttl

Add cloak_ttl for plugin_cloak. With default lik 600, it would solve all need adequately currently visible in example-cloaking-rules.txt.

And allow to set per record ttl in cloaking-rules.txt. As if you are testing you might want to use 0 and localhost could use like 86400 or greater. But that is another patch and I'll leave it to somebody else to implement when they have need to have something like that.

But anyways, most other caching dns servers default cache_min_ttl to 0 and recommend not to increase it. And it would be great to even provide some cache for blocked requests, or your log is then full of them.